pycode/memilio-simulation/memilio/simulation/abm.cpp

```diff
@@ -69,7 +69,7 @@ PYBIND11_MODULE(_simulation_abm, m)
         .value("PublicTransport", mio::abm::LocationType::PublicTransport)
         .value("TransportWithoutContact", mio::abm::LocationType::TransportWithoutContact);
 
-    py::class_<mio::abm::TestParameters>(m, "TestParameters")
+    pymio::bind_class<pymio::PickleFlag::TryPickling, mio::abm::TestParameters>(m, "TestParameters")
```

Comment:
I think types like this should be ForcePickling. Otherwise someone adds a parameter that isn't serializable and the pickling disappears. I think TryPickling only makes sense for generic types.